shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/conf/ct/SyReplicaTable.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (C) 2016-2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.shunting.yard.replicator.exec.conf.ct;
+
+import org.hibernate.validator.constraints.NotBlank;
+
+public class SyReplicaTable {
+  private @NotBlank String databaseName;
```

Comment:
I think this can be blank or null or whatever, same for tableName

Reply:
I think we agreed that for now, the feature where a user can override either DB or table is not supported. We will observe the usage of this feature and based on that improvise coz the usecase can be that "Put all selected tables in one DB".

Reply:
I don't remember agreeing on that. it's trivial to make it optional

Reply:
Its done.